### ISSUE TITLE
the theme isn't working

### ISSUE THREAD

**Issue by real-nickel** [pH7Software/pH7-Social-Dating-CMS#1142]
Hi,

I have installed ph7 to my web host via softaculous, and when i tried it, the css doesn't match the one on the demo. it has 0 fonts and i believe there is no css at all... am i doing something wrong?

![image](https://github.com/pH7Software/pH7-Social-Dating-CMS/assets/51296698/1f519a60-c242-4edc-8849-e3c3f1ed30e2)

**Comment by TheAlgoslingers** [comment]
No, my client reported same issue. You aren't wrong. The developers are wrong with the http routing configuration.

**Comment by real-nickel** [comment]
ok, thanks for telling.

**Comment by real-nickel** [comment]
wait so is this issue soon being fixed or...?

**Comment by TheAlgoslingers** [comment]
I dont think its fixed

**Comment by nick-nomad** [comment]
Same problem, is this still not fixed?

**Comment by pH-7** [comment]
Hi there,

It looks to be an insure with your web server configuration and/or with the htaccess for your specific case. 
Please also ensure the "mod_rewrite" module is correctly setup on your end.
https://ph7builder.com/doc/en/how-to-install-rewrite-module
